pdr_backend/cli/arg_threshold.py

```diff
@@ -0,0 +1,93 @@
+from typing import List, Union
+
+from enforce_typing import enforce_types
+
+
+class ArgThreshold:
+    def __init__(self, threshold_str: str):
+        """
+        @arguments
+          threshold_str -- e.g. "vb_2100" "db_2090.5" "tb_208"
+        """
+        prefix, value_str = _unpack_threshold_str(threshold_str)
+
+        if prefix not in ["vb", "db", "tb"]:
+            raise ValueError("threshold should start with vb, db or tb")
+
+        self.prefix = prefix  # 'db', 'vb', 'tb'
+        self.value_str = value_str
+        self.value = float(self.value_str)
+
+    def __eq__(self, other):
+        return str(self) == str(other)
+
+    def __str__(self):
+        return self.prefix + "_" + self.value_str
+
+    def threshold(self) -> float:
+        return self.value
+
+
+class ArgThresholds(List[ArgThreshold]):
+    def __init__(self, thresholds: Union[List[str], List[ArgThreshold]]):
+        if not isinstance(thresholds, list):
+            raise TypeError("hresholds must be a list")
+
+        thres = []
+        for thre in thresholds:
```

Comment:
you said earlier we only have one type of threshold in a list of thresholds. If so, maybe also add a check that all the arg thresholds in a list should have the same type

Reply:
open this up for discussions, I think it can be a useful use case: "binance BTC/USDT ETH/USDT 5m tb_405,vb_200" which maps BTC/USDT tb_405 and ETH/USDT vb_200" previous consideration was to forcefull seperate different thresholds